### ISSUE TITLE
Key type

### ISSUE THREAD

**Issue by brainsiq** [tcs-de/nodecache#16]
Is it intentional that it has to be string? It's not mentioned in the docs but couldn't get the cache to work with an integer key.

**Comment by varunpatro** [comment]
^bump

**Comment by mpneuried** [comment]
Hi, internally the key is used as an object key.

The data object will look like:
```js
data = {
  "yourkey": {
    "v": "the cached value",
    "t": 0 // the ttl
  }
}
```

This is the reason why a integer is not possible.
I created a separate [Issue](#48) to validate the key's type.

Also see [this](https://github.com/tcs-de/nodecache/blob/master/_src/lib/node_cache.coffee#L159) code position

I hope this helps ;-)

**Comment by mpneuried** [comment]
Will be fixed by #48
